packages/frontend-v2/utils/theme/index.ts

```diff
@@ -48,16 +48,13 @@ const colors = {
     },
   },
   neutral: {
-    main: "#e7ebf1",
-    50: "#eff2f6",
-    100: "#d2d7de",
-    200: "#b3bcc8",
-    300: "#f1f5fa",
+    gray0: "#f4f6f9",
```

Comment:
Can we get different names for this? Either create a new type under neutral and then add weights to them. Eg instead of `neutral.gray0` it would be `neutral.gray.100 neutral.gray.200` etc